crates/release-automation/examples/fix-json.rs

```diff
@@ -27,7 +27,10 @@ fn get_clippy_output() -> Result<PathBuf, Error> {
     cmd.args(&[
         "clippy",
         "--target-dir",
-        &format!("{}/clippy", env!("CARGO_TARGET_DIR")),
+        &format!(
+            "{}/clippy",
+            option_env!("CARGO_TARGET_DIR").unwrap_or("target")
```

Comment:
according to https://doc.rust-lang.org/cargo/reference/environment-variables.html this env var should be set by `cargo`. i didn't envision running this without `cargo` but i'd like to learn if you are and why.

Reply:
@steveeJ the docs are misleading, it says it "sets" the env variable, but I don't think it literally does that.

Running `cargo run --example fix-json` without this change produces:

```
error: environment variable `CARGO_TARGET_DIR` not defined
  --> crates/release-automation/examples/fix-json.rs:30:31
   |
30 |         &format!("{}/clippy", env!("CARGO_TARGET_DIR")),
   |                               ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `env` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
that's odd. is `CARGO_BUILD_TARGET_DIR` set?

Reply:
no, that's not set either.

Reply:
that's odd as well. ideally we'd have code that looks for the cargo workspace root to use its target directory.